src/utils/date-utils-test.js

```diff
@@ -63,3 +67,26 @@ describe('convertPotentialTimestampToSeconds', () => {
     );
   });
 });
+
+describe('convertPotentialTimestampToMilliseconds', () => {
```

Comment:
Please also update these tests so they are not reimplementing the logic being verified.

Reply:
Sorry, which logic are they reimplementing? I thought they are not related to the bug I fixed?

Reply:
Line 73 here is just copied from line 63 of the implementation, for example.

Reply:
Would you these tests to use literal values?

Reply:
Yes, that would be the straightforward way to remove the logic.